setup.cfg

```diff
@@ -27,6 +27,7 @@ package_dir =
     bumble = bumble
     bumble.apps = apps
     bumble.apps.link_relay = apps/link_relay
+    bumble.transport = bumble/transport
```

Comment:
I don't think this is needed, because `transport` is already a subdir of `bumble`. I realize that this also means that line 29 above is also likely not needed (but line 28 is needed, because `apps` isn't a subdir)

Reply:
Indeed is not needed, I'll remove it (in the internal version line 26 was missing `bumble.transport` in `package`s directive and I just carried the explicit package_dir over).

Reply:
Removed.